common/persistence/tests/postgresql_test.go

```diff
@@ -28,592 +28,14 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/suite"
-
-	persistencetests "go.temporal.io/server/common/persistence/persistence-tests"
-	"go.temporal.io/server/common/persistence/serialization"
-	"go.temporal.io/server/common/persistence/sql"
-	"go.temporal.io/server/common/persistence/sql/sqlplugin"
-	_ "go.temporal.io/server/common/persistence/sql/sqlplugin/postgresql"
-	sqltests "go.temporal.io/server/common/persistence/sql/sqlplugin/tests"
-	"go.temporal.io/server/common/resolver"
 )
 
-func TestPostgreSQLShardStoreSuite(t *testing.T) {
-	testData, tearDown := setUpPostgreSQLTest(t)
-	defer tearDown()
-
-	shardStore, err := testData.Factory.NewShardStore()
-	if err != nil {
-		t.Fatalf("unable to create PostgreSQL DB: %v", err)
-	}
-
-	s := NewShardSuite(
-		t,
-		shardStore,
-		serialization.NewSerializer(),
-		testData.Logger,
-	)
-	suite.Run(t, s)
-}
```

Comment:
Can you move the code back to `postgresql_test.go` from `postgresql_suite.go`? I don't think there's any need to separate them.